extension/vsc-ts/howdoi/src/extension.ts

```diff
@@ -0,0 +1,136 @@
+import * as vscode from 'vscode';
```

Comment:
This seems to be a copy of the file from above. Is it possible to just have one version?